utils/node_local_test.py

```diff
@@ -2147,6 +2151,67 @@ def test_pydaos_kv(server, conf):
     daos._cleanup()
     log_test(conf, pydaos_log_file.name)
 
+def run_daos_test(server, conf):
+    """Run the daos_test program to check for errors
+
+    Note that this doesn't check the result of daos_test itself, but
+    so far simply checks the logs for generated errors.
+    """
+
+    env = get_base_env()
+    env['DAOS_AGENT_DRPC_DIR'] = server.agent_dir
+    env['POOL_SCM_SIZE'] = '1'
+    env['POOL_NVME_SIZE'] = '0'
+    env['OMPI_MCA_btl'] = 'self,tcp'
+    env['OMPI_MCA_oob'] = '^ud,ucx'
+    env['OMPI_MCA_pml'] = '^ucx'
+    log_file = tempfile.NamedTemporaryFile(prefix='dnt_test_',
+                                           suffix='.log',
+                                           delete=False)
+    env['D_LOG_FILE'] = log_file.name
+
+    daos_test_bin = os.path.join(conf['PREFIX'], 'bin', 'daos_test')
+
+    rc = subprocess.run([daos_test_bin, '--help'],
```

Comment:
(pylint-subprocess-run-check) Using subprocess.run without explicitly set `check` is not recommended.